app/views/comments/_comment.html.erb

```diff
@@ -28,6 +28,9 @@
         <div>
           <%= comment.user.name %>
           <%= comment.body %>
+          <% if comment.user.id == current_user.id %>
```

Comment:
How would you feel about including a policy object here?

Also, you could likely use `commen.user_id`.

Reply:
This has been addressed in #225